src/Controller/CustomerController.php

```diff
@@ -55,38 +68,52 @@ public function addCustomer(Request $request): JsonResponse
      */
     public function getOneCustomer($id): JsonResponse
     {
-        $customer = $this->customerRepository->findOneBy(['id' => $id]);
-
-        $data = [
-            'id' => $customer->getId(),
-            'firstName' => $customer->getFirstName(),
-            'lastName' => $customer->getLastName(),
-            'email' => $customer->getEmail(),
-            'phoneNumber' => $customer->getPhoneNumber(),
-        ];
 
-        return new JsonResponse(['customer' => $data], Response::HTTP_OK);
-    }
+        try {
 
-       /**
-     * @Route("", name="get_all_customers", methods={"GET"})
-     */
-    public function getAllCustomers(): JsonResponse
-    {
-        $customers = $this->customerRepository->findAll();
-        $data = [];
+            $customer = $this->customerRepository->findOneBy(['id' => $id]);
 
-        foreach ($customers as $customer) {
-            $data[] = [
+            if ($customer == null) {
```

Comment:
Can make use of builtin methods like `is_null()` whenever possible.

Reply:
i have replaced with is_null() now